packages/user-activity-broadcaster/src/glance-cobrowse-spript.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable unicorn/prefer-dom-node-dataset */
+
+import envVar from '@availity/env-var/src'
```

Comment:
You shouldn't have to add the `/src`. Does it not work without it?

Reply:
src removed